openmc/material.py

```diff
@@ -560,6 +560,73 @@ def add_element(self, element, percent, percent_type='ao', enrichment=None):
         for nuclide in element.expand(percent, percent_type, enrichment):
             self.add_nuclide(*nuclide)
 
+    def add_material(self, material, percent, percent_type='ao'):
+        """Mix a material to the material
+        Parameters
+        ----------
+        material : openmc.Material
+            Material to add
+        percent : float
+            Atom or weight percent
+        percent_type : {'ao', 'wo'}
+            'ao' for atom percent and 'wo' for weight percent
+        """
+
+        if self._macroscopic is not None:
+            msg = 'Unable to add a Material to Material ID="{0}" as a ' \
```

Comment:
Numbered format is not necessary here:
```suggestion
            msg = 'Unable to add a Material to Material ID="{}" as a ' \
```
Same thing below